libs/net/traffic_generator.py

```
@@ -186,3 +189,41 @@ def _ensure_is_running(self) -> bool:
 def is_tcp_connection(server: TcpServer, client: BaseTcpClient) -> bool:
     return server.is_running() and client.is_running()
+
+
+@contextlib.contextmanager
+def client_server_active_connection(
+    client_vm: BaseVirtualMachine,
+    server_vm: BaseVirtualMachine,
+    spec_logical_network: str,
+    port: int = IPERF_SERVER_PORT,
+    maximum_segment_size: int = 0,
+) -> Generator[tuple[VMTcpClient, TcpServer], None, None]:
```

Ruff (0.14.10)

Unnecessary default type arguments

Remove default type arguments

(UP043)